src/components/ContractName.tsx

```diff
@@ -41,20 +38,10 @@ function ContractNameSuspended({
   clearType,
   network,
 }: ContractNameProps) {
-  const { emailDerivativeContracts = [], ERC721derivativeContracts = [] } =
-    useSnapshot(SealCredStore)
-  // We will always get a string
-  let contractName = getContractName(address, network) as string
+  let contractName = getContractName(address, network, truncate)
 
-  if (clearType) {
-    if (contractName && emailDerivativeContracts.includes(address))
-      contractName = contractName.replace(' email', '')
-
-    if (contractName && ERC721derivativeContracts.includes(address))
-      contractName = contractName.replace(' (derivative)', '')
-  }
-
-  contractName = prettifyContractName(contractName, truncate)
+  if (clearType)
```

Comment:
Actually, maybe we should use `clearType` directly inside `getContractName` function?